coaster/sqlalchemy/statemanager.py

```diff
@@ -204,7 +218,7 @@ def redraft(self):
 from ..signals import coaster_signals
 from .roles import RoleMixin
 
-__all__ = ['StateManager', 'StateTransitionError',
+__all__ = ['StateManager', 'StateTransitionError', 'AbortTransitionError',
```

Comment:
Since the abort exception is not passed through to the caller, it’s not an error. This should be named just `AbortTransition`.